Framework/Core/src/DataProcessingDevice.cxx

```diff
@@ -526,7 +526,7 @@ void DataProcessingDevice::doPrepare(DataProcessorContext& context)
         while (true) {
           FairMQParts parts;
           result = fairMQChannel.Receive(parts, 0);
```

Comment:
This already returns int64_t: https://github.com/FairRootGroup/FairMQ/blob/v1.4.26/fairmq/FairMQChannel.h#L309

Reply:
Agreed. The problem was an int on DPL side.